scripts/ibmcloud_post_install.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+cat > /root/install_complete.sh <<EOF
+#!/usr/bin/env bash
+set -euo pipefail
+
+export HOME=/root
+
+### Status flag
+LOGFILE=/tmp/ibmcloud-post-install.log
+log () {
+  echo "\$(date) +++ \$1" >> \$LOGFILE
+}
+
+log "Starting ibmcloud-post-install.sh"
+
+{
+### Base prerequisites
+dnf install -y git
+
+### Get AI repo
+cd /tmp
```

Comment:
maybe better to clone it to /home ?

Reply:
i mean why to remove it? if i understand right those machines will be used for test-infra in any case right?